src/hyperion/experiment_plans/wait_for_robot_load_then_centre_plan.py

```diff
@@ -118,6 +118,11 @@ def wait_for_robot_load_then_centre_plan(
             set_energy_composite,
         )
 
+    actual_energy_ev = 1000 * (yield from bps.rd(composite.dcm.energy_in_kev))
+    parameters.hyperion_params.ispyb_params.current_energy_ev = actual_energy_ev
```

Comment:
Should: A better way to do this is just to read it when we read all the ispyb parameters during the data collection inside `read_hardware_for_ispyb_during_collection`

Reply:
I have questions about the witchcraft that happens in that method!